app/components/shared/inputs/MediaGalleryInput.vue.ts

```diff
@@ -0,0 +1,49 @@
+import { shell } from 'electron';
+import Component from 'vue-class-component';
+import { Inject } from 'util/injector';
+import { Prop } from 'vue-property-decorator';
+import { BaseInput } from './BaseInput';
+import { IInputMetadata } from './index';
+import { MediaGalleryService } from 'services/media-gallery';
+import { TextInput } from './inputs';
+import FormGroup from './FormGroup.vue';
+
+interface IMediaGalleryMetadata extends IInputMetadata {
+  fileName: string;
+  clearImage: string;
+}
```

Comment:
After last refactoring it's better to add this interface to `\app\components\widgets\inputs\index.ts`. Since these interfaces can be used without importing the component file.